<?xml version="1.0"?>
<launch>
    <!-- 启动左电机驱动 -->
    <node name="xian_dj_car_chassis_base_electric_left_wheel" pkg="xian_dj_car_chassis_control_pkg" type="xian_dj_car_chassis_base_electric" output="screen">
        <param name="server_ip" value="192.168.1.132" />
        <param name="server_port" value="8886" />
        <param name="pub_topic_name" value="xian_dj_car_chassis_base_electric_left_state_msg" />
        <param name="sub_topic_name" value="xian_dj_car_chassis_diff_driver_control_left_write_velocity_msg" />
    </node>

    <!-- 启动右电机驱动 -->
    <node name="xian_dj_car_chassis_base_electric_right_wheel" pkg="xian_dj_car_chassis_control_pkg" type="xian_dj_car_chassis_base_electric" output="screen">
        <param name="server_ip" value="192.168.1.133" />
        <param name="server_port" value="8886" />
        <param name="pub_topic_name" value="xian_dj_car_chassis_base_electric_right_state_msg" />
        <param name="sub_topic_name" value="xian_dj_car_chassis_diff_driver_control_right_write_velocity_msg" />
    </node>

    <node name ="xian_dj_car_chassis_diff_driver_control" pkg="xian_dj_car_chassis_control_pkg" type="xian_dj_car_chassis_diff_driver_control" />
    <node name ="xian_dj_car_chassis_error_monitor" pkg="xian_dj_car_chassis_control_pkg" type="xian_dj_car_chassis_error_monitor" />


    <!-- rosrun xian_dj_car_chassis_control_pkg xian_dj_car_chassis_base_electric _server_ip:="192.168.1.132" _server_port:=8886 _motor_driver:="left_driver" -->
    <!-- rosrun xian_dj_car_chassis_control_pkg xian_dj_car_chassis_base_electric _server_ip:="192.168.1.133" _server_port:=8886 _motor_driver:="right_driver" -->
    <!-- <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch">
    </include> -->
</launch>